android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -259,6 +257,21 @@ public void setIconView(final int tag)
         }
     }
 
+    public void onSetProps()
+    {
+        if ( hasAddUnitIdUpdated.compareAndSet( true, false ) )
```

Comment:
Same comment as above - can we just do `if ( !TextUtils.isEmpty( adUnitId ) )` instead of maintaining a new state `hasAddUnitIdUpdated`?